### PR TITLE
Adds the CMO's stamp to their locker

### LINKED ISSUE #50419: Metastation CMO spawns without a stamp in his office.
## Round ID: NaN
(Did not discover this issue from playing a specific round on tgstation hosted servers)

## Testmerges:

#49065: 'Disable library machine uis' by optimumtact at commit f261fc0b45

## Reproduction:

Join round as CMO, hark, there's no fucking stamp in your office as there should be, generally speaking all the heads are meant to have stamps AFAIK

image

### PR DESCRIPTION
**Pull request by TheVekter**
## About The Pull Request

Fixes #50419 

## Changelog
:cl:
add: Added the CMO's stamp to their locker.
/:cl:

### CODE REVIEW

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -28439,6 +28436,9 @@
 /obj/structure/window/reinforced{
 	dir = 4
 	},
+/obj/machinery/computer/rdconsole/core{
```
Comment (trollbreeder) [inline _maps/map_files/BoxStation/BoxStation.dmm:28439]: Undocumented change

**Comment by trollbreeder** [comment]
There's quite a few undocumented changes related to replacing the AI fixer consoles with RnD consoles. Did you branch out of master correctly?

**Comment by TheVekter** [comment]
Haha fuck no I'm fixing it

**Comment by TheVekter** [comment]
Fixing this in a better manner.

**Pull request closed without merging**